docs/spec/expression-user-send.zh-CN.md

```
@@ -20,68 +20,94 @@ title: 表达｜用户发送
 
 ### 编辑发送
 
-用户在发送信息之后，若发现存在错误或需补充内容时，可以对已发送气泡进行再次编辑修改，提供更为便捷灵活的交互体验，使用户能够更好地表达自身意图。
+<ImagePreview>
+<img class="preview-img no-padding" src="https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*-uiSQraZL-UAAAAAAAAAAAAADgCCAQ/fmt.webp">
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)